apps/avifgainmaputil/imageio.cc

```diff
@@ -163,6 +163,11 @@ avifResult ReadAvif(avifDecoder* decoder, const std::string& input_filename,
   }
   if (ignore_profile) {
     avifRWDataFree(&decoder->image->icc);
+#if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
```

Comment:
Delete this. If this file is compiled, `AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP` must be defined.

Reply:
Ha yes good point!